src/Blueprint/Blueprint/Cmdlets/BlueprintDefinition/ImportAzureRMBlueprint.cs

```diff
@@ -49,6 +49,8 @@ public class ImportAzureRmBlueprint : BlueprintDefinitionCmdletBase
         #region Cmdlet Overrides
         public override void ExecuteCmdlet()
         {
+            RegisterBlueprintRp(SubscriptionId ?? DefaultContext.Subscription.Id);
```

Comment:
@filizt why is this needed?

Reply:
@cormacpayne import cmdlet makes a PUT call to create a blueprint and it could be the case that user hasn't registered the RP with the subscription yet. This call is making sure Blueprint service is registered.